xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -515,16 +515,19 @@
         <summary>Schedules a cancel operation on this <see cref="T:System.Threading.CancellationTokenSource" /> after the specified time span.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[
-            The countdown for the delay starts during this call.  When the delay expires, 
-            this <xref:System.Threading.CancellationTokenSource> is canceled, if it has
-            not been canceled already.
-            
-            Subsequent calls to `CancelAfter` will reset the delay for this <xref:System.Threading.CancellationTokenSource>, if it has not been canceled already.
+
+## Remarks
+ The countdown for the delay starts during this call. When the delay expires, this <xref:System.Threading.CancellationTokenSource> is canceled, if it has not been canceled already.
+
+ Subsequent calls to `CancelAfter` will reset the delay for this <xref:System.Threading.CancellationTokenSource>, if it has not been canceled already.  
+  
+> [!NOTE]
+>  Valid `TimeSpan` objects can be created with a larger range of values than accepted by this method. Calling `CancelAsync` with `TimeSpan delay` values greater than `TimeSpan.FromMilliseconds(Int32.MaxValue)` (or `TimeSpan.FromMilliseconds(UInt32.MaxValue - 1)` on some versions of .NET) up to `TimeSpan.MaxValue` will throw `System.ArgumentOutOfRangeException`.
```

Comment:
```suggestion
> It's possible to create valid `TimeSpan` objects that have a larger range of values than is accepted by this method. If you call `CancelAfter` with a `TimeSpan` value greater than `TimeSpan.FromMilliseconds(Int32.MaxValue)` (or `TimeSpan.FromMilliseconds(UInt32.MaxValue - 1)` on some versions of .NET) up to `TimeSpan.MaxValue`, an `System.ArgumentOutOfRangeException` is thrown.
```

Reply:
Actually this has become too wordy overall, simplified it. The focus in the later part should be on the specific parameter `delay` than the type, which is already know in params section.